packages/web-app-admin-settings/src/components/AppTemplate.vue

```diff
@@ -10,6 +10,7 @@
               id="admin-settings-breadcrumb"
               class="oc-flex oc-flex-middle"
               :items="breadcrumbs"
+              :represents-folder-structure="false"
```

Comment:
I really hope that this prop is not needed, see other comment 🙈  But if we need it, I just realized here, that `folder-structure` doesn't fit as wording. Breadcrumbs can represent any hierarchy, e.g. `Users > Horst` if we'd decide to have a view for single users in the future.